test/test_connection.py

```diff
@@ -43,6 +49,12 @@ def test_match_hostname_mismatch(self):
             )
             self.assertEqual(e._peer_cert, cert)
 
+    def test_recent_date(self):
+        # This test is to make sure that the RECENT_DATE value
+        # doesn't get too far behind what the current date is.
```

Comment:
Let's add a comment here expressly saying what has to be done to fix this test when it fails.